content/zh/style-guide/_index.md

```diff
@@ -0,0 +1,104 @@
+---
+title: 风格指南
+toc_hide: true
+menu:
+main:
+weight: 10
+pre: <i class='fas fa-ruler-horizontal'></i>
+---
```

Comment:
I think this part will bring build failure :)

Reply:
yes, I've fixed its format.